drivers/axi_core/axi_dmac/axi_dmac.c

```diff
@@ -71,6 +123,77 @@ int32_t axi_dmac_write(struct axi_dmac *dmac,
 	return SUCCESS;
 }
 
+/***************************************************************************//**
+ * @brief axi_dmac_transfer_nonblock
+ *******************************************************************************/
+int32_t axi_dmac_transfer_nonblock(struct axi_dmac *dmac,
```

Comment:
Let's rename this from "nonblock" to "nonblocking" for consistency with the rest of the functions (e.g., "uart_read_nonblocking", "uart_write_nonblocking").